tox.ini

```diff
@@ -203,6 +206,9 @@ deps =
     application_celery: celery<6.0
     application_celery-py{py37,37}: importlib-metadata<5.0
     application_gearman: gearman<3.0.0
+    component_sklearn-scikitlearnlatest: scikit-learn
+    component_sklearn-scikitlearn101: scikit-learn < 1.1
+    component_sklearn-scikitlearn020: scikit-learn < 0.21
```

Comment:
Use 2 digits for each tox version to match convention. `0101` and `0020`. Makes it slightly easier to read.